pkg/util/admission/elastic_cpu_work_handle.go

```diff
@@ -117,6 +120,32 @@ func (h *ElasticCPUWorkHandle) overLimitInner() (overLimit bool, difference time
 	return false, h.differenceWithAllottedAtLastCheck
 }
 
+// NoteWorkPreempted record the fact that this handle going over limit preempted
+// work at least once.
+func (h *ElasticCPUWorkHandle) NoteWorkPreempted() {
+	h.preemptedWork = true
+}
+
+// NoteWorkPreempted record the fact that this handle going over limit preempted
+// work at least once.
+func (h *ElasticCPUWorkHandle) NoteWorkPaced(delay time.Duration) {
+	h.pacedWork += delay
+}
+
+// PacedWork returns whether or not work using this handle reported being
+// preempted due to Overlimit (NB: this may be false even if `Overlimit()` was
+// called and returned true, in the case that the caller elected to ignore it
+// and continue working rather than be preempted).
+func (h *ElasticCPUWorkHandle) PreemptedWork() bool {
+	return h.preemptedWork
+}
+
+// PacedWork returns the total amount of time spent being paced that any work
+// using this handle reported.
+func (h *ElasticCPUWorkHandle) PacedWork() time.Duration {
```

Comment:
Just checking that you've intentionally not called this anywhere and are leaving that for a future patch?